blog/2023/10-01-a-call-for-developers/index.mdx

```diff
@@ -0,0 +1,51 @@
+
+Will you join us?
+
+<!--truncate-->
+
+We keep a close eye on the buzz about Jellyfin around Reddit and the Internet more broadly; in the last couple weeks we have seen a few major Plex-related kerfufles and the comments inevitably include the following exchange:
+
+    Person A: Use Jellyfin instead
+    Person B: Jellyfin sucks, it lacks client X or feature Y
+    Person C: Works for me
+    Person D: It needs polish and more apps
+
+Sometimes one of us will jump in, but I think it's high time for us to make a public request for something that's been on our minds as a team for quite a while.
+
+For those who are not aware, Jellyfin is a bit unusual, even within the FLOSS world. We are *explicitly anti-commercial*. Our team is entirely composed of volunteers who contribute because they like what we're doing and can help out in whatever ways they can. This includes developers, translators, and community members who help moderate the community and assist others in troubleshooting and working through problems.
+
+No one gets paid for Jellyfin work, by design. We're not backed by any company or "organization" with its own agenda. We have no "monetization plan" or anything of the sort, and actively avoid bug bounties. Jellyfin isn't a "product" in the commercial sense of the word. And while we do take donations, these *only* cover our infrastructure costs and the rare piece of developer client hardware as needed; we don't pay developers out of donations, and never will. Money is not an issue (for what we use it for), and that's not really what we need. This is a choice we've made from day 1 to avoid Jellyfin following in the footsteps of nearly every other player in this space, who start out as FLOSS, start adding user-hostile features in the name of money, and then go closed source. We want to be different. Want to be volunteer-only and working on this thing to help make media streaming available to everyone.
+
+But, the team behind Jellyfin is under strain. We love what we do, and want to keep doing it. But our core contributor base is actually fairly small, at most about 30 active people, for everything - the core server, the webUI, and all of our clients. And it hasn't really grown in quite some time. Some of our developers have been with us since the early days, some joined later, but life sometimes gets in the way, and we've lost some over the years, so the number has been pretty flat for the last 3 years all together. For some parts of the project, especially individual apps, we have at most 1 or 2 people working on them, for free in their own time, because it "scratches their itch".
+
+Some very-well-request features/clients have gotten *no* traction at all, with no one coming in wanting to develop them. We've had to abandon some (like Chromecast for a while, though it's getting new life in the last few weeks) because of this.
```

Comment:
```suggestion
Some very-well-requested features/clients have gotten *no* traction at all, with no one coming in willing to start/help developing them. We've had to abandon some (like Chromecast for a while, though it's getting new life in the last few weeks) because of this.
```